requirements.txt

```diff
@@ -1,6 +1,5 @@
 coverage~=7.6.4
 flake8~=7.1.1
```

Comment:
Sorry, I should have put this in the ticket @ojokure. I think flake8 itself could also be moved to pyproject.toml since it is also only a dev dependency.